app/Models/Status.php

```diff
@@ -32,6 +32,7 @@ class Status extends Model
         'business'   => Business::class,
         'visibility' => StatusVisibility::class,
         'event_id'   => 'integer',
+        'tweet_id'   => 'integer',
```

Comment:
I'm not sure how good PHP handles the uint_64 translation to integer. [Twitter recommends](https://developer.twitter.com/en/docs/twitter-ids) to use strings here to avoid [implicit conversions to Float](https://www.php.net/manual/en/language.types.integer.php) which are used in PHP.